presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1484,4 +1493,59 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
   return nodeStatus;
 }
 
+void PrestoServer::convertToVeloxPlan(
+    proxygen::HTTPMessage* message,
+    proxygen::ResponseHandler* downstream,
+    const std::vector<std::unique_ptr<folly::IOBuf>>& body) {
+  protocol::ExecutionFailureInfo failure;
+  try {
+    auto headers = message->getHeaders();
+
+    std::string planFragmentJson = util::extractMessageBody(body);
+    protocol::PlanFragment planFragment = json::parse(planFragmentJson);
+
+    auto queryCtx = core::QueryCtx::create();
+    VeloxInteractiveQueryPlanConverter converter(queryCtx.get(), pool_.get());
```

Comment:
We could have a local Memory Pool for this code. Can avoid using the global server pool.